dev/com.ibm.ws.microprofile.metrics.2.x.monitor_fat/publish/servers/TestEDF2/bootstrap.properties

```diff
@@ -1,2 +1,2 @@
 bootstrap.include=../testports.properties
-com.ibm.ws.logging.trace.specification=*=info:com.ibm.ws.microprofile.metrics.monitor.*=all:com.ibm.ws.webcontainer.monitor.WebContainerMonitor=all:com.ibm.websphere.monitor.meters=all
\ No newline at end of file
+com.ibm.ws.logging.trace.specification=*=info:com.ibm.ws.microprofile.metrics.monitor.*=all:com.ibm.ws.webcontainer.monitor.WebContainerMonitor=all:com.ibm.websphere.monitor.meters=all:com.ibm.ws.webcontainer.servlet=all
```

Comment:
I usually have `.*` at the end of a trace string so I can see the traces from sub-classes too.  e.g. `com.ibm.ws.webcontainer.servlet.*=all`.

Reply:
the `ServletWrapper` class is using just the package name as the logger:
https://github.com/OpenLiberty/open-liberty/blob/integration/dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/servlet/ServletWrapper.java#L127

Reply:
However, I did have success getting the classes of a package from the before set spec
`com.ibm.ws.webcontainer.monitor.WebContainerMonitor=all:com.ibm.websphere.monitor.meters=all`